packages/react-server-cli/src/compileClient.js

```diff
@@ -181,7 +181,6 @@ function packageCodeForBrowser(entrypoints, outputDir, outputUrl, hot, minify, l
 				{
 					test: /\.css$/,
 					loader: extractTextLoader,
```

Comment:
Please do not forget to remove the comma as well, we don't know when it breaks :-)